web/wp-content/themes/dctx-v2/inc/customizer/customizer.php

```diff
@@ -0,0 +1,108 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-fsp6to.php, but found phpcs-scan-fSP6tO.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).